hs_core/views/resource_folder_hierarchy.py

```diff
@@ -714,16 +714,16 @@ def data_store_create_folder(request):
     folder_path = request.POST.get('folder_path', None)
 
     try:
-        folder_path = _validate_path(folder_path, 'folder_path')
+        folder_path = _validate_path(folder_path)
     except ValidationError as ex:
         return HttpResponse(str(ex), status=status.HTTP_400_BAD_REQUEST)
 
     try:
         create_folder(res_id, folder_path)
     except SessionException as ex:
         return HttpResponse(ex.stderr, status=status.HTTP_500_INTERNAL_SERVER_ERROR)
-    except DRF_ValidationError as ex:
-        return HttpResponse(ex.detail, status=status.HTTP_400_BAD_REQUEST)
+    except (DRF_ValidationError, SuspiciousFileOperation) as ex:
+        return HttpResponse(str(ex), status=status.HTTP_400_BAD_REQUEST)
```

Comment:
Do we want to change the parsing on this message so that it isn't a raw dict/json? Maybe this is consistent with our messaging and I didn't know that...
![image](https://user-images.githubusercontent.com/17934193/203146520-86aaba25-61d9-49d3-9141-52115d483d6f.png)

Reply:
It's a larger issue that needs work both in the backend and front-end. May be we should create a separate issue to fix it.

Reply:
sounds good thanks